### PR TITLE
Update workload.md

### PR DESCRIPTION
**Pull request by Ghosgt**

### CODE REVIEW

File: content/zh/docs/reference/glossary/workload.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ For example, a workload that has a web server and a database might run the
 database in one {{< glossary_tooltip term_id="StatefulSet" >}} and the web server
 in a {{< glossary_tooltip term_id="Deployment" >}}.
 -->
-代表不同类型或部分工作负载的各种核心对象包括 DaemonSet， Deployment， Job， ReplicaSet， and StatefulSet。
+代表不同类型或部分工作负载的各种核心对象包括 DaemonSet，Deployment，Job，ReplicaSet and StatefulSet。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/workload.md:45]:
```suggestion
代表不同类型或部分工作负载的各种核心对象包括 DaemonSet、Deployment、Job、ReplicaSet 和 StatefulSet。
```

**Review by Arhell — COMMENTED** [review]
Pls sign the CLA in order for the PR to be reviewed.
more info
https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by jimangel** [comment]
Hey there! @Ghosgt, looks like you haven't signed the CLA yet. Could I please have you do that? https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by tengqm** [comment]
Missing CLA for 30+ days. Please feel free to reopen and resume this work. Thanks.
/close

**Pull request closed without merging**